package.json

```diff
@@ -310,6 +369,47 @@
         "language": "sas",
         "path": "./snippets/function-snippets.json"
       }
+    ],
+    "viewsContainers": {
+      "activitybar": [
+        {
+          "id": "sas-view-container",
+          "title": "SAS",
+          "icon": "icons/sas-svg.svg"
```

Comment:
we usually need light and dark icons respectively. Will this fit both light and dark themes?

Reply:
I originally thought this would be fine, because the activity bar is usually a dark color. That said, it looks like themes can change the activity bar color. With that in mind, I'm wondering if you can provide dark and light icons, @skyeaussie ? Or point me to where I can find these?

Reply:
After further exploration, it looks like vscode natively implements some kind of bitmasking. As it stands, the icon is black. However, here's how it shows up in vscode. 
![Screenshot 2023-02-06 at 1 32 27 PM](https://user-images.githubusercontent.com/1058265/217055890-f6df1c4c-1186-420c-9b1e-e7a026b6bdb9.png)

In this case, I _think_ what we have is sufficient

Reply:
cool